sdks/java/io/clickhouse/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://s.apache.org/beam-owners
+
+reviewers:
+  - kanterov
```

Comment:
I put my name here, but I'm not a beam committer. How do we find the owner? @iemejia